payment/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,55 @@
+# Translation of the participant app
+# Copyright (C) 2020 Simon Ellmann
+# This file is distributed under the same license as the registration package.
```

Comment:
witch license ?

GPL3+ or MIT or is this not jet decided?

Reply:
I'd prefer AGPL but this should be discussed first

Reply:
I'll open a own issue for this ...